tests/baselines/reference/errorOnInitializerInObjectType.errors.txt

```diff
@@ -0,0 +1,17 @@
+tests/cases/compiler/errorOnInitializerInObjectType.ts(2,17): error TS1246: An object type property cannot have an initializer.
+tests/cases/compiler/errorOnInitializerInObjectType.ts(6,17): error TS1246: An object type property cannot have an initializer.
+
+
+==== tests/cases/compiler/errorOnInitializerInObjectType.ts (2 errors) ====
+    interface Foo {
+        bar: number = 5;

```

Comment:
I'm surprised that we don't go off the rails here since you're not advancing the parser at all (i.e. you use `lookAhead`, not `tryParse`).